NEWS.md

```diff
@@ -1,3 +1,11 @@
+# proxyC 0.4.0
+
```

Comment:
If `$RCPP_PARALLEL_NUM_THREADS` no longer works, it should be mentioned in the NEWS.md as such.

```suggestion

## Breaking change

- Number of threads used during parallellization should now be set with `options(proxyC.threads = ...)` instead of the environment variable `RCPP_PARALLEL_NUM_THREADS` 
```

However, if we provide backwards compatibility with `RCPP_PARALLEL_NUM_THREADS`, then this doesn't need to be added to `NEWS.md`.